src/Mod/Draft/draftguitools/gui_polararray.py

```diff
@@ -126,19 +124,70 @@ def click(self, event_cb=None):
             # of the calling class (this one).
             self.ui.accept()
 
+    def add_center_callbacks(self):
+        """Execute when the center selection should be enabled"""
+        self.callback_move = \
+            self.view.addEventCallbackPivy(self.location, self.move)
+        self.callback_click = \
+            self.view.addEventCallbackPivy(self.mouse_event, self.click)
+
+    def remove_center_callbacks(self):
+        """Execute when the center selection should be disabled"""
+        if hasattr(self, "callback_move"):
+            self.view.removeEventCallbackPivy(self.location,
+                                      self.callback_move)
+            del(self.callback_move)
```

Comment:
Here the purpose is that I check for the existence of the callback with `hasattr` before so I won't try to remove the callback if it is already removed. Alternatively I could initialize the callback attribute to `Ǹone` and just set it to None after removing it so I could then use `if self.callback_move` instead of `hasattr`.

Reply:
Yes, but I mean, why run `del` at all?

Reply:
No particular reason, so you would say initializing the attribute to `None` and then using and `if self.callback_move:` would be  idiomatic/best practice?

Reply:
Yes, in general you should always initialize the variables explicitly. This makes it easier for other programmers to recognize what will be used and what won't.